src/pages/ResetPage/resetPage.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Text } from "components/Text";
+import { Input } from "components/Input";
+import { supabaseClient } from "../../config/supabase-client";
+import ResetStyles from "./reset.module.css";
+
+const ResetPage = () => {
+  const navigate = useNavigate();
+  const handleNavigate20 = () => navigate("/");
+  const handleNavigate19 = () => navigate("/loginpage");
```

Comment:
Same suggestion as with the other `handleNavigatexx` constants -- try to make it more descriptive or just directly substitute the expression in place of the `handleNavigatexx`s